pkg/pillar/cmd/volumemgr/#sizemgmt.go#

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2020 Zededa, Inc.
```

Comment:
I guess this file should not be part of the commit?

Reply:
Fixed